src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -551,6 +551,22 @@ static grpc_lb_addresses *process_serverlist_locked(
   return lb_addresses;
 }
 
+static grpc_lb_addresses *process_fallback_backend_addresses_locked(
+  grpc_exec_ctx *exec_ctx, const grpc_lb_addresses *fallback_backend_addresses) {
```

Comment:
You'll need to fix the indentation here in order to get the clang-format sanity check to pass.